tools/proofers/preview.js

```diff
@@ -689,6 +704,49 @@ $(function () {
             }
         }
 
+        // check that math delimiters \[ \], \( \) are correctly matched
+        function checkMath() {
+            var mathRe = /\\\[|\\\]|\\\(|\\\)/g;
+            var result;
+            var tag, openTag = null;
+            var openStart = 0;
+            while (true) {
```

Comment:
infinite loops scare me in case the break is never hit. Would it be possible to rework or somehow guarantee we will always break?

Reply:
I suppose we could run the loop a large number of times (1000?) and if it doesn't "break" before this show a warning message.

Reply:
If you are convinced that it isn't necessary I'm not opposed to infinite loops, I just can't convince myself by looking at this without investing some more time.